GPflowOpt/acquisition.py

```diff
@@ -456,3 +458,42 @@ def __mul__(self, other):
             return AcquisitionProduct(self.operands.sorted_params + other.operands.sorted_params)
         else:
             return AcquisitionProduct(self.operands.sorted_params + [other])
+
+
+class MCMCAcquistion(AcquisitionSum):
+    def __init__(self, acquisition, n_slices, **kwargs):
+        assert isinstance(acquisition, Acquisition)
+        assert n_slices > 0
+
+        copies = [copy.deepcopy(acquisition) for _ in range(n_slices - 1)]
+        for c in copies:
+            c._optimize_restarts = 0
+
+        super(MCMCAcquistion, self).__init__([acquisition] + copies)
```

Comment:
I could just copy n_slices times, but then some of the inherited tests fail (when calling `models`, the models passed on an AcquisitionAggregation are expected to be returned. So it makes to do it this way.